Library/Homebrew/test/cleanup_spec.rb

```diff
@@ -69,6 +69,28 @@
     end
   end
 
+  specify "::update_disk_cleanup_size" do
+    shutup do
+      described_class.instance_eval("@disk_cleanup_size = 0")
+      described_class.update_disk_cleanup_size(128)
+    end
+    expect(described_class.instance_variable_get("@disk_cleanup_size")).to eq(128)
```

Comment:
I'm not sure it's worth verifying internal class variables are set; test the externally visible functionality instead (which may mean not having tests for every method)